packages/zowe-explorer/src/generators/icons/index.ts

```diff
@@ -31,6 +31,7 @@ export enum IconId {
     "folder" = "folder",
     "folderOpen" = "folderOpen",
     "migrated" = "migrated",
+    "imperativeError" = "imperativeError",
```

Comment:
Should the icon be called "error" instead of "imperativeError"? Sorry if I'm being nitpicky, but the "imperativeError" seems like it may be too specific as we may want to use the icon for other error types in the future 🙂